### DISCUSSION TITLE
Change of local IP Address

### DISCUSSION THREAD

**Discussion by chuchischef** [teslamate-org/teslamate#2861] [Q&A] [answered]
Hi

I changed my local IP-Address range. The dashboard is working just fine.
But, e.g the TeslaMate Button next to the Dashboards drop down is still using the old IP-Address.
Also if I want to add charging costs, the link points to the old IP. As it seems, everything on port 4000 is still using the old IP.
Where can I change that setting?

Thanks for the help

/Roman

**Comment by cwanja** [comment, marked as answer]
Did you adjust the IP settings in TeslaMate? Might also take a restart of the container.

**Reply by chuchischef** [reply]
I couldn't find any settings...
Where can I find them?

**Reply by cwanja** [reply]
TeslaMate webpage > (hamburger menu) Settings > URLs (bottom of the page)

**Reply by chuchischef** [reply]
Thanks a lot. That solved my problem.

Cheers
